addons/stock/models/stock_picking.py

```diff
@@ -813,31 +813,26 @@ def _get_overprocessed_stock_moves(self):
 
     @api.multi
     def _create_backorder(self, backorder_moves=[]):
-        """ Move all non-done lines into a new backorder picking. If the key 'do_only_split' is given in the context, then move all lines not in context.get('split', []) instead of all non-done lines.
+        """ Move all non-done lines into a new backorder picking.
         """
-        # TDE note: o2o conversion, todo multi
         backorders = self.env['stock.picking']
         for picking in self:
             backorder_moves = backorder_moves or picking.move_lines
```

Comment:
I have the feeling that `backorder_moves` has become useless here, especially if it is used inside a loop on the recordset.